bundles/org.openhab.binding.salus/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="salus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Salus Binding</name>
+	<description>
+		This is the binding for Salus, a renowned manufacturer of smart home solutions. This binding allows
+		integration of Salus' wide range devices into your home control system, facilitating bidirectional communication
+		and
+		control.
+	</description>
+
```

Comment:
please add ```connection``` according to https://next.openhab.org/docs/developer/addons/addon.html
cloud? or is can it be operated also locally?

Reply:
can bridged be detected in the network? -> binding suggestion possible? maybe you want to use the new addon suggestion finders and provide date here...

Reply:
Added `connection` tag

> can bridged be detected in the network? 

Nope. You need to create bridge manually (username and pass cannot be inferred). 

> new addon suggestion finders

any doc?

Reply:
as long as there is not device on the local network which can be detected (e.g. a physical bridge device, or a heat pump connected to the network), there is noting to do....

Its not about inferring the login data etc, it is about suggesting a plugin to be installed.
https://next.openhab.org/docs/developer/addons/addon.html#discovery-methods

Reply:
OK, now I understand. Sadly salus plugin cannot look for devices in local network. It's only able to connect to cloud and the suggest the devices